diagnostic.md

```diff
@@ -6,21 +6,23 @@ Record your responses inside the fenced code blocks below each question.
     and what are the main task(s) you perform inside an Ember Route?
 
     ```md
-    <!-- your response here -->
+    The router identifies (and assigns custom URLs to) the various routes. The routes themselves define the data and behavior to be loaded when that route is loaded.
     ```
 
 1.  What is the command to generate a route named `boston` nested under
     `campus`?
 
     ```md
-    <!-- your response here -->
+    ember g route campus/boston
     ```
 
 1.  Suppose you have a nested route at the URL `/campus/boston`. How would you
     use the `link-to` helper to generate an appropriate link?
 
     ```md
-    <!-- your response here -->
+    You might add something like
+    {{#link-to 'campus'}}Back to All Campuses{{/link-to}}
+    in your template file under /campus/boston
```

Comment:
To link to /campus/boston, we'd want to use `{{#link-to 'campus.boston'}}Boston{{/link-to}}`. That's the appropriate answer if we were to link just to 'campus', though